examples/dynamic-config-cp/resource.go

```diff
@@ -27,8 +27,8 @@ import (
 	hcm "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/http_connection_manager/v3"
 
 	"github.com/envoyproxy/go-control-plane/pkg/cache/types"
-	"github.com/envoyproxy/go-control-plane/pkg/cache/v3"
-	"github.com/envoyproxy/go-control-plane/pkg/resource/v3"
+	_ "github.com/envoyproxy/go-control-plane/pkg/cache/v3"
+	_ "github.com/envoyproxy/go-control-plane/pkg/resource/v3"
```

Comment:
i think this still fails CI (apologies if you fixed somehow)

if it is - then i think just revert it and only worry about `gofmt` linting in this PR

Reply:
Yeah, thanks, I also noticed this failure. I have fixed it in a new commit, using their package names.

I tested it locally, and the go-control-plane compiles ok now.